src/EnergyPlus/ElectricPowerServiceManager.cc

```diff
@@ -1939,6 +1960,10 @@ namespace EnergyPlus {
 			generatorType = GeneratorType::pV;
 			compGenTypeOf_Num = DataGlobalConstants::iGeneratorPV;
 			compPlantTypeOf_Num = DataPlant::TypeOf_PVTSolarCollectorFlatPlate;
+				} else if ( InputProcessor::SameString( objectType, "Generator:PVWatts" ) ) {
```

Comment:
Something went awry here on the indentation it appears.

Reply:
Yep. Thought I caught all of those.